usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/db/OracleConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package com.oracle.cloud.world.demo.db;
```

Comment:
Suggested to remove Cloud World reference (generalize)...

Reply:
Renamed